src/js/GlobalConstants.js

```diff
@@ -4,7 +4,7 @@
  */
 
 const globalConstants = {
-    API: process.env.API,
+    API: process.env.USASPENDING_API,
```

Comment:
Should this be called `USASPENDING_API` to match dev.config and prod.config?

Reply:
The only place we're directly using `process.env.USASPENDING_API` is here. If we change the `globalConstants.API` to `globalConstants.USASPENDING_API`we'd have to update like 100 files.